docs/setup.md

```diff
@@ -4,20 +4,20 @@
 # Installation
 
 1. To install a solution pack, click **Content Hub** > **Discover**.
-2. From the list of solution pack that appears, search for and select **Impossible Traveller Threat Response**.
-3. Click the **Impossible Traveller Threat Response** solution pack card.
-4. Click **Install** on the bottom to begin installation.stallation
+2. From the list of solution pack that appears, search for **Impossible Traveler Threat Response**.
+3. Click the **Impossible Traveler Threat Response** solution pack card.
+4. Click **Install** on the lower part of the screen to begin installation.
 
 ## Prerequisites
 
-|Solution Pack Name|Purpose|
-| :- | :- |
-|SOAR Framework|Required for Incident Response modules|
-|SOC Simulator|Required for Scenario Module and SOC Simulator connector|
+| Solution Pack Name | Version | Purpose                                                  |
+|:-------------------|:--------|:---------------------------------------------------------|
+| SOAR Framework     | v2.2.1  | Required for Incident Response modules                   |
```

Comment:
There is no specific SOAR Framework version requirement for this SP, it can work on lower version as well

Reply:
However, we have to keep this table consistent with similar other tables in other SP documentation. So please specify a minimum version with which this SP will work

Reply:
You can give v2.0.2